packages/assets-controllers/src/assetsUtil.test.ts

```diff
@@ -0,0 +1,248 @@
+import { NetworksChainId } from '@metamask/network-controller';
```

Comment:
Another case where this change is actually brought over from `main` (this isn't really part of this PR).